cita-network/src/synchronizer.rs

```diff
@@ -459,3 +452,74 @@ impl Synchronizer {
         self.block_lists = self.block_lists.split_off(&height);
     }
 }
+
+#[derive(Clone)]
+pub struct SynchronizerClient {
+    sender: Sender<SynchronizerMessage>,
+}
+
+impl SynchronizerClient {
+    pub fn new(sender: Sender<SynchronizerMessage>) -> Self {
+        SynchronizerClient { sender }
+    }
+
+    pub fn handle_local_status(&self, msg: SynchronizerMessage) {
+        self.send_msg(msg);
+    }
+
+    pub fn handle_remote_status(&self, msg: SynchronizerMessage) {
+        self.send_msg(msg);
+    }
+
+    pub fn handle_remote_response(&self, msg: SynchronizerMessage) {
+        self.send_msg(msg);
+    }
+
```

Comment:
These three functions work for the same thing?

Reply:
Yes, I want to distinguish between different semantics for caller.